Day-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,24 @@
 import React from 'react';
 import './Button.css';
+
+const ActionButton = () => {
+    return (
+        <div className="action-buttons">
+            <div className="clear">clear</div>
+            <div className="numbers">
+                <div className="number">7</div>
```

Comment:
Each of these number buttons should actually be a rendered component at the App.js level, or the button container level, not individual headers inside this component.